### ISSUE TITLE
libpmempool_backup/TEST6 randomly fails on appveyor

### ISSUE THREAD

**Issue by igchor** [pmem/issues#972]
Log: [libpmempool_backup6.txt](https://github.com/pmem/issues/files/2534585/libpmempool_backup6.txt)

There were some attempts to debug this issue, however the cause was not found.

Ref:
https://github.com/pmem/issues/issues/715
https://github.com/pmem/issues/issues/603

**Comment by marcinslusarz** [comment]
This is stil an issue. https://ci.appveyor.com/project/pmem/pmdk/builds/22118211/job/3e8l1f3fplv2ayc6
Relevant fragment:
```
<libpmempool>: <3> [libpmempool.c:345 pmempool_checkU] 
<libpmempool>: <3> [check.c:197 check_step] 
<libpmempool>: <3> [check_backup.c:384 check_backup] backup_path C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.set_backup
<libpmempool>: <3> [check_backup.c:275 backup_poolset_overwrite] 
<libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 000001F68AA8BCB8 spart 000001F68AA67428
<libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
<libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 000001F68AA8BD38 spart 000001F68AA674A8
<libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
<libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 000001F68AA8BDB8 spart 000001F68AA67528
<libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part3_backup"
<libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 000001F68AA8BE38 spart 000001F68AA675A8
<libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
<libpmempool>: <1> [pool.c:797 pool_set_part_copy] dmapped < smapped: dmapped = 20971519, smapped = 20971520
<libpmempool>: <1> [pool.c:798 pool_set_part_copy] assertion failure: 0
```

**Comment by marcinslusarz** [comment]
Another case: https://ci.appveyor.com/project/pmem/pmdk/builds/23509934/job/78d3gy116cox77v3
```
[00:19:59] <libpmempool>: <3> [set.c:1255 util_poolset_set_size] pool size set to 83689472
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
[00:19:59] <libpmempool>: <3> [file.c:258 util_file_get_size] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
[00:19:59] <libpmempool>: <3> [file.c:223 util_file_get_type] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
[00:19:59] <libpmempool>: <3> [file.c:258 util_file_get_size] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
[00:19:59] <libpmempool>: <3> [file.c:223 util_file_get_type] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part3_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
[00:19:59] <libpmempool>: <3> [file.c:258 util_file_get_size] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
[00:19:59] <libpmempool>: <3> [file.c:223 util_file_get_type] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
[00:19:59] <libpmempool>: <3> [libpmempool.c:345 pmempool_checkU] 
[00:19:59] <libpmempool>: <3> [check.c:197 check_step] 
[00:19:59] <libpmempool>: <3> [check_backup.c:384 check_backup] backup_path C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.set_backup
[00:19:59] <libpmempool>: <3> [check_backup.c:275 backup_poolset_overwrite] 
[00:19:59] <libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 0000026CE330F868 spart 0000026CE33074E8
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part1_backup"
[00:19:59] <libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 0000026CE330F8E8 spart 0000026CE3307568
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part2_backup"
[00:19:59] <libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 0000026CE330F968 spart 0000026CE33075E8
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part3_backup"
[00:19:59] <libpmempool>: <3> [pool.c:748 pool_set_part_copy] dpart 0000026CE330F9E8 spart 0000026CE3307668
[00:19:59] <libpmempool>: <3> [file.c:131 util_file_exists] path "C:\temp\\test_libpmempool_backup6ðŸ˜˜â â â ™â …É—PMDKÓœâ¥ºðŸ™‹\pool.part4_backup"
[00:19:59] <libpmempool>: <1> [pool.c:797 pool_set_part_copy] dmapped < smapped: dmapped = 20971519, smapped = 20971520
[00:19:59] <libpmempool>: <1> [pool.c:798 pool_set_part_copy] assertion failure: 0
[00:19:59] <libpmempool>: <3> [libpmempool.c:85 libpmempool_fini] 
[00:19:59] <libpmempool>: <3> [mmap.c:101 util_mmap_fini] 
```

**Comment by marcinslusarz** [comment]
This is finally fixed. It will be part of 1.7 and 1.6.1. It's also committed to stable-1.5 and stable-1.4 branches, so if we'll decide to release from those branches the fix will be in 1.5.2 and/or 1.4.3.
